components/gfx/display_list/mod.rs

```diff
@@ -976,6 +985,7 @@ pub enum DisplayItem {
     StackingContextClass(Arc<StackingContext>),
     LayeredItemClass(Box<LayeredItem>),
     NoopClass(Box<BaseDisplayItem>),
+    IframeClass(Box<IframeDisplayItem>),
```

Comment:
@mrobinson Sounds good - is it fine to this as a follow up once this PR lands do you think?